### PR TITLE
Feat dynamic stepper:Added the stepper functionality and dynamically changing the Status

### PR DESCRIPTION
**Pull request by Amartyakumar**

### CODE REVIEW

File: pages/apps/_app/event/_id/index.vue [hunk 1]
```diff
@@ -1036,8 +1054,36 @@ export default {
       }
     },
   },
-
   methods: {
+    updateStepper() {
+      const status = this.eventData.Status
+      console.log('status1', status)
```
Comment (SukantGujar) [inline pages/apps/_app/event/_id/index.vue:1060]: Remove

File: pages/apps/_app/event/_id/index.vue [hunk 2]
```diff
@@ -1036,8 +1054,36 @@ export default {
...
+          `${url}Events/${this.$route.params.id}`,
+          {
+            Status: statusName,
+          }
+        )
+        if (res) {
+          this.refresh()
+        }
+      } catch (e) {
+        console.log('Error', e)
```
Comment (SukantGujar) [inline pages/apps/_app/event/_id/index.vue:1084]: Add more context. What operation was tried? What were the inputs?

File: pages/event/_id/index.vue [hunk 3]
```diff
@@ -681,6 +689,11 @@ export default {
   mixins: [configLoaderMixin],
   data() {
     return {
+      // ready: false,
```
Comment (SukantGujar) [inline pages/event/_id/index.vue:692]: Remove

File: pages/event/_id/index.vue [hunk 4]
```diff
@@ -689,9 +702,53 @@ export default {
...
+        status: {},
       },
     }
   },
+  computed: {
+    isReady() {
+      const status = this.data.event.Status
+      console.log('status1', status)
+      if (
+        status === 'Not ready' ||
```
Comment (SukantGujar) [inline pages/event/_id/index.vue:714]: Simply return this expression. No need for if.

File: pages/event/_id/index.vue [hunk 5]
```diff
@@ -689,9 +702,53 @@ export default {
...
+        status === 'Registration closed'
+      ) {
+        return true
+      } else return false
+    },
+
+    isSoldOut() {
+      const status = this.data.event.Status
+      console.log('status3', status)
+      if (status === 'Sold out' || status === 'Registration closed') {
```
Comment (SukantGujar) [inline pages/event/_id/index.vue:739]: Same here and in the below if statement.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: pages/event/_id/index.vue (hunk 3 above)
Comment (Amartyakumar) [reply pages/event/_id/index.vue:692]: This file is not in use at the moment.This branch was created earlier

File: pages/event/_id/index.vue (hunk 4 above)
Comment (Amartyakumar) [reply pages/event/_id/index.vue:714]: This file is not in use now

File: pages/event/_id/index.vue (hunk 5 above)
Comment (Amartyakumar) [reply pages/event/_id/index.vue:739]: This file is not in use

Comment (SukantGujar) [reply pages/event/_id/index.vue:739]: Then delete this file please.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**